CPAC/resources/configs/pipeline_config_default.yml

```diff
@@ -1334,6 +1333,13 @@ functional_preproc:
     # Apply functional mask in native space
     apply_func_mask_in_native_space: On
 
+  template_space_func_masking:
+    run: Off
+      # Anatomical_Resampled: Resample anatomical brain mask in standard space to get BOLD brain mask in standard space. Adapted from DCAN Lab's BOLD mask method from the ABCD pipeline. ("Create fMRI resolution standard space files for T1w image, wmparc, and brain mask […] don't use FLIRT to do spline interpolation with -applyisoxfm for the 2mm and 1mm cases because it doesn't know the peculiarities of the MNI template FOVs")
+    using: [Anatomical_Resampled]
+
+    apply_func_mask_in_template_space: Off
```

Comment:
I (think I) get `"run"` is generating the mask and `"apply_func_mask_in_template_space"` is applying the generated mask, but I think we could be clearer if we add some comments in the default and blank configs and/or naming the keys something like `"generate"` and `"apply"`

Reply:
If we don't want / care to allow for generating-but-not-applying, then yeah, I think just one switch makes sense. Otherwise, I just think
1. it's not clear that `run` generates but doesn't apply, and
2. `apply_func_mask_in_template_space` is unnecessarily long.

But those are just my opinions. I think with some comments saying what the options do, these keys can work fine.